packages/app/src/elements/settings-security.ts

```diff
@@ -258,7 +268,7 @@ export class SettingsSecurity extends StateMixin(Routing(LitElement)) {
         ) {
             return;
         }
-        await app.api.revokeSession(id);
+        await app.api.revokeSession({ id });
```

Comment:
I wasn't aware [we were "limited" by `process` to only send one argument](https://github.com/padloc/padloc/blob/main/packages/core/src/server.ts#L236-L240), so I had to change this into an object, once I realized that.